src/rma/conf/base.py

```diff
@@ -368,3 +368,4 @@
 CELERY_TASK_SOFT_TIME_LIMIT = 600
 
 ZAKEN_PER_TASK = 10
+ZAKEN_PER_QUERY = 1000000
```

Comment:
A cool Python 3.6 feature is that you can use underscores in literals to improve readability:

```suggestion
ZAKEN_PER_QUERY = 1_000_000
```